playbooks/engineer-workflow.md

```diff
@@ -49,26 +49,38 @@ PR titles, because by default, we squash our commits when merging a PR, and this
 automatically ends up using the conventional commit format. Using the prefix in your branch name is not essential,
 but it's, well, nice.
 
-### Commits
+### Commit Messages
 
 > A well-crafted git commit message is the best way to communicate context about a change to fellow developers (and
 > indeed to [our] future selves). -[How to write a git commit message](https://chris.beams.io/posts/git-commit/)
 
-The code change itself should be internally consistent and coherent. If you're modifying a method signature,
-propagate the change to calling code. If you've implemented a new behavior, update test assertions to match. Except
-in rare cases where you intend to share a work-in-progress, commits should not leave tests in a failing state.
+In support of this, Artsy engineering adopted [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) as its commit message convention (see RFC [Best Practices for Naming and Merging PRs](https://github.com/artsy/README/issues/327)). We ecourage you to read the [spec](https://www.conventionalcommits.org/en/v1.0.0/#specification) for more detail but, in general, all commits should match the following format ([see examples](https://excalidraw.com/#json=dMGWGEi9PAmxEZXBsC0kG,kUHekoAEfwndFljUuTUqOA)): 
 
-Commit subjects should concisely describe the enhancement or fix in the imperative form ("Add tracking of email
-deliveries" or "Undo home page A/B test"). Git subcommands (like `git blame`, `revert`, `log`...) expect this
-format and become more transparent to use when you adhere to that convention.
+```
+<type>[optional scope]: <description>
```

Comment:
**suggestion:** some folks at Artsy include the Jira ticket number in their PR title (which, of course, ends up being the squashed commit) because we have things (Peril, I think?) set up to automatically perform the issue linking. Could we mention and codify this? Might actually belong in the PRs section below, but I think it would be nice to call out.

Some recent examples from me:

```
feat(FX-4093): curated marketing collections

chore[FX-3378]: deprecate addToCalendar helper function

chore[FX-3378]: stop using Cohesion addToCalendar helper
```

Reply:
Yeah, that's a good call! I will add an Artsy-specific example and include that the ticket id can be used included in the `optional scope`, which will create the automatic jira link.